src/dagmc/DagMC.hpp

```diff
@@ -381,17 +387,23 @@ class DagMC {
   ErrorCode get_root(EntityHandle vol_or_surf, EntityHandle& root);
 
   /** Get the instance of MOAB used by functions in this file. */
+  [[ deprecated("Replaced by std::shared_ptr<Interface> moab_instance_ptr()") ]]
   Interface* moab_instance() {return MBI;}
+  std::shared_ptr<Interface> moab_instance_ptr() {return MBI_shared_ptr;}
```

Comment:
This method is also a little inconsistent with our current design. What if the deprecated constructor is used and this object is a `nullptr`?

Reply:
sounds good !